core/controllers/admin.py

```diff
@@ -204,6 +213,43 @@ def _reload_collection(self, collection_id):
         else:
             raise Exception('Cannot reload a collection in production.')
 
+    def _generate_dummy_explorations(
+            self, num_dummy_exps_to_generate, num_dummy_exps_to_publish):
+        """
+        Generates and Publishes given number of dummy explorations.
+
+        Args:
+            num_dummy_exps_to_generate: int. Count of dummy explorations to
+                be generated.
+            num_dummy_exps_to_publish: int. Count of explorations to
+                be published.
+
+        Raises:
+            Exception: Environment is not DEVMODE.
+        """
+
+        if feconf.DEV_MODE:
+            logging.info(
+                '[ADMIN] %s generated %s number of dummy explorations' %
+                (self.user_id, num_dummy_exps_to_generate))
+            possible_titles = ['Hulk Neuroscience', 'Quantum Starks',
+                               'Wonder Anatomy',
+                               'Elvish, language of "Lord of the Rings',
+                               'The Science of Superheroes']
+            for i in range(num_dummy_exps_to_generate):
+                title = random.choice(possible_titles)
+                category = random.choice(feconf.SEARCH_DROPDOWN_CATEGORIES)
+                new_exploration_id = exp_services.get_new_exploration_id()
+                exploration = exp_domain.Exploration.create_default_exploration(
+                    new_exploration_id, title=title, category=category,
+                    objective='Dummy Objective')
+                exp_services.save_new_exploration(self.user_id, exploration)
+                if i <= (num_dummy_exps_to_publish - 1):
```

Comment:
You don't need the parentheses. (In Python we try to avoid parens where possible.)

Also, might perhaps be cleaner to do `i < num_dummy_exps_to_publish`, but your call on that one.